pkg/plugins/repo/version.go

```diff
@@ -44,7 +44,7 @@ func SelectSystemCompatibleVersion(log log.PrettyLogger, versions []Version, plu
 		}, nil
 	}
 	for _, v := range versions {
-		if v.Version == version {
+		if normalizeVersion(v.Version) == version {
```

Comment:
should we need normalize both in the comparison?

Reply:
Yep, `version` was already normalized, that's the bug:

https://github.com/grafana/grafana/blob/2c230c34d1d5f38bc744fc621df37d06757664a6/pkg/plugins/repo/version.go#L25